docs/Validation-and-Serialization.md

```diff
@@ -124,7 +124,7 @@ The route validation internally relies upon [Ajv](https://www.npmjs.com/package/
 Validating the input is very easy: just add the fields that you need inside the route schema, and you are done!
 
 The supported validations are:
-- `body`: validates the body of the request if it is a POST or a PUT.
+- `body`: validates the body of the request if it is a POST, PUT or PATCH method, and doesn't work with GET and HEAD methods.
```

Comment:
```suggestion
- `body`: validates the body of the request if it is a POST, PUT, or PATCH method.
```
Second half of the sentence is extraneous, if a HTTP request method is not mentioned in the first half of the sentence then it should be assumed that it is not supported.